src/Chapter1.hs

```diff
@@ -619,8 +636,14 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int
+sumLast2 n = fDigit (mod (abs n) 100) + secDigit (abs n)
+  where
+    fDigit :: Int -> Int
+    fDigit x = (abs x) `quot` 10
```

Comment:
`quot` is usually used in pair with `rem`, while `mod` with div` 🙂 

Also, you can use `n` directly in here, as it is in scope 🙂 

If you pass `abs n` to this function, you can safely skip `abs` step in the function 🙂

Reply:
```suggestion
sumLast2 n = fDigit n + secDigit n
  where
    fDigit :: Int -> Int
    fDigit n = rem (abs n) 100 `quot` 10
   
    secDigit :: Int -> Int
    secDigit n = rem (abs n) 10
```

@vrom911 Thank you 🙏🏼 for such a detailed review. 

I've replaced `mod` with `rem` and passed `abs n` to the `fDigit` and `secDigit` directly. 🙂

> Also, you can use ` n` directly in here, as it is in scope 🙂

I'm having trouble following this; the code works fine once I relace `x` with `n` but it triggers this warning:
> This binding for ‘n’ shadows the existing binding bound at ../../learn4haskell/src/Chapter1.hs:635:10

Reply:
Oh yes, sorry! This looks great already with `x` as well, but I noticed that you can use `n` from the above directly, and in that case, you won't need to take any `Int` at all. Here is how it may look like:

```haskell
sumLast2 n = fDigit + secDigit
  where
    fDigit :: Int
    fDigit = (mod (abs n) 100) `quot` 10

    secDigit :: Int
    secDigit = rem (abs n) 10
...
```

you see how I directly use `n` inside those functions, instead of passing the argument to it 🙂 

This is just a note, your solution looks great already! Just FYI :)

Reply:
Wow, the function looks way more cleaner now. Thank you 🙂